component/common/src/main/java/org/wso2/carbon/identity/smsotp/common/constant/Constants.java

```diff
@@ -101,6 +103,7 @@
         private final String code;
         private final String message;
         private final String description;
+        private String remainingAttempts;
```

Comment:
Can't we use int for this?

Reply:
Resolved with 2a64f0f0c44301e1fd4887100400e50c346706ab